extensions/cli/geoserver/src/main/java/mil/nga/giat/geowave/cli/geoserver/GeoServerAddStyleCommand.java

```diff
@@ -42,6 +46,9 @@
 	private String stylesld = null;
 
 	@Parameter(description = "<GeoWave style name>")
+	@RestParameters(names = {
```

Comment:
are `@RestParameters` used at all anymore - I don't think so, I think the annotation should probably be deleted and removed everywhere